psalm.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="7"
```

Comment:
```suggestion
    errorLevel="1"
```

The reported error count is the same.
```
------------------------------
143 errors found
------------------------------
9521 other issues found.
You can display them with --show-info=true
------------------------------
Psalm can automatically fix 3982 of these issues.
Run Psalm again with 
--alter --issues=InvalidReturnType,MissingReturnType,MissingClosureReturnType,InvalidNullableReturnType,InvalidFalsableReturnType,MissingParamType --dry-run
to see what it can fix.
------------------------------
```

Reply:
If the count is the same then why not go with the higher level? That way when we integrate with the pipeline we will catch more issues.

Reply:
> Level 1 is the most strict, level 8 is the most lenient.
https://psalm.dev/docs/running_psalm/error_levels/

Reply:
Ah, 7 is too loose.